stripe-ui-core/src/main/java/com/stripe/android/uicore/image/ImageLruDiskCache.kt

```diff
@@ -77,6 +80,20 @@ class ImageLruDiskCache(
         }
     }
 
+    private fun compressFormatFromUrl(url: String): CompressFormat {
+        return when {
+            url.endsWith("png") -> CompressFormat.PNG
```

Comment:
is it save to assume there are only 3 types? should we check png, webp, jpg and throw an exception otherwise?

Reply:
Good idea, that makes more sense - https://github.com/stripe/stripe-android/pull/5763/commits/74ccc8bc5e3810d98435170497392ccc1e63b1b2